_assets/js/netlify/topics/generateChildElements.js

```diff
@@ -0,0 +1,30 @@
+const notAllowed = ['em', 'strong', 'u', 'i', 'b', 'ul', 'blockquote', 'table', 'thead', 'td', 'tr','tfooter', 'tbody', 'br', 'input'];
+
+const generateChildElements = (html) =>
+  html.map((item) => {
```

Comment:
this is a `map`, but not all code paths return a value.

this will lead to undefined values in the array - consider adding a filter before the map:

```
html.filter(item => {
  if (item.children && item.children.length > 0) return true;
  if (notAllowed.includes(item.nodeName.toLowerCase()) return false;
  return true;
})
```